packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
> if you can already mock out the MethodChannel, what's the value of
> also wrapping the API into a container class?

If just gives the caller two ways to test (mocking out the class or mocking out the channel), depending on where they want to hook into things.  That said, I don't have a strong opinion on it.  @mehmetf WDYT?